From 670ad70989eff14b9c5892555bbc612b01569661 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Tue, 5 Jul 2016 10:17:25 +0800
Subject: [PATCH] ARM64: dts: rockchip: Rename OPP nodes as opp@<opp-hz>

It would be better to name OPP nodes as opp@<opp-hz> as that will ensure
that multiple DT nodes don't contain the same frequency. Of course we
expect the writer to name the node with its opp-hz frequency and not any
other frequency.

And that will let the compile error out if multiple nodes are using the
same opp-hz frequency.

Change-Id: Icefba93f7a95752e344b5a092a83931bf4d1e682
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3366.dtsi      | 22 +++++------
 arch/arm64/boot/dts/rockchip/rk3368.dtsi      | 18 ++++-----
 .../boot/dts/rockchip/rk3399-evb-rev2.dtsi    | 37 ++++++++++---------
 arch/arm64/boot/dts/rockchip/rk3399.dtsi      | 28 +++++++-------
 4 files changed, 53 insertions(+), 52 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3366.dtsi b/arch/arm64/boot/dts/rockchip/rk3366.dtsi
index 783caac064db..3ae404c56cd4 100644
--- a/arch/arm64/boot/dts/rockchip/rk3366.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3366.dtsi
@@ -131,29 +131,29 @@
 		compatible = "operating-points-v2";
 		opp-shared;
 
-		opp00 {
+		opp@408000000 {
 			opp-hz = /bits/ 64 <408000000>;
 			opp-microvolt = <950000>;
 			clock-latency-ns = <40000>;
 			opp-suspend;
 		};
-		opp01 {
+		opp@600000000 {
 			opp-hz = /bits/ 64 <600000000>;
 			opp-microvolt = <950000>;
 		};
-		opp02 {
+		opp@816000000 {
 			opp-hz = /bits/ 64 <816000000>;
 			opp-microvolt = <1000000>;
 		};
-		opp03 {
+		opp@1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
 			opp-microvolt = <1075000>;
 		};
-		opp04 {
+		opp@1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
 			opp-microvolt = <1175000>;
 		};
-		opp05 {
+		opp@1296000000 {
 			opp-hz = /bits/ 64 <1296000000>;
 			opp-microvolt = <1250000>;
 		};
@@ -1673,23 +1673,23 @@
 		compatible = "operating-points-v2";
 		opp-shared;
 
-		opp00 {
+		opp@96000000 {
 			opp-hz = /bits/ 64 <96000000>;
 			opp-microvolt = <1100000>;
 		};
-		opp01 {
+		opp@192000000 {
 			opp-hz = /bits/ 64 <192000000>;
 			opp-microvolt = <1100000>;
 		};
-		opp02 {
+		opp@288000000 {
 			opp-hz = /bits/ 64 <288000000>;
 			opp-microvolt = <1100000>;
 		};
-		opp03 {
+		opp@375000000 {
 			opp-hz = /bits/ 64 <375000000>;
 			opp-microvolt = <1125000>;
 		};
-		opp04 {
+		opp@480000000 {
 			opp-hz = /bits/ 64 <480000000>;
 			opp-microvolt = <1200000>;
 		};
diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index 058fd911288a..132765e7129d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -207,25 +207,25 @@
 		compatible = "operating-points-v2";
 		opp-shared;
 
-		opp00 {
+		opp@408000000 {
 			opp-hz = /bits/ 64 <408000000>;
 			opp-microvolt = <1200000>;
 			clock-latency-ns = <40000>;
 			opp-suspend;
 		};
-		opp01 {
+		opp@600000000 {
 			opp-hz = /bits/ 64 <600000000>;
 			opp-microvolt = <1200000>;
 		};
-		opp02 {
+		opp@816000000 {
 			opp-hz = /bits/ 64 <816000000>;
 			opp-microvolt = <1200000>;
 		};
-		opp03 {
+		opp@1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
 			opp-microvolt = <1200000>;
 		};
-		opp04 {
+		opp@1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
 			opp-microvolt = <1200000>;
 		};
@@ -235,21 +235,21 @@
 		compatible = "operating-points-v2";
 		opp-shared;
 
-		opp00 {
+		opp@408000000 {
 			opp-hz = /bits/ 64 <408000000>;
 			opp-microvolt = <1200000>;
 			clock-latency-ns = <40000>;
 			opp-suspend;
 		};
-		opp01 {
+		opp@600000000 {
 			opp-hz = /bits/ 64 <600000000>;
 			opp-microvolt = <1200000>;
 		};
-		opp02 {
+		opp@816000000 {
 			opp-hz = /bits/ 64 <816000000>;
 			opp-microvolt = <1200000>;
 		};
-		opp03 {
+		opp@1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
 			opp-microvolt = <1200000>;
 		};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
index 57b16076efa7..d58b8406cc7f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
@@ -56,71 +56,72 @@
 };
 
 &cluster0_opp {
-	opp00 {
+	opp@408000000 {
 		opp-hz = /bits/ 64 <408000000>;
 		opp-microvolt = <800000>;
 		clock-latency-ns = <40000>;
 	};
-	opp01 {
+	opp@600000000 {
 		opp-hz = /bits/ 64 <600000000>;
 		opp-microvolt = <800000>;
 	};
-	opp02 {
+	opp@816000000 {
 		opp-hz = /bits/ 64 <816000000>;
 		opp-microvolt = <800000>;
 	};
-	opp03 {
+	opp@1008000000 {
 		opp-hz = /bits/ 64 <1008000000>;
 		opp-microvolt = <875000>;
 	};
-	opp04 {
+	opp@1200000000 {
 		opp-hz = /bits/ 64 <1200000000>;
 		opp-microvolt = <925000>;
 	};
-	opp05 {
+	opp@1416000000 {
 		opp-hz = /bits/ 64 <1416000000>;
 		opp-microvolt = <1025000>;
 	};
-	opp06 {
+	opp@1512000000 {
 		opp-hz = /bits/ 64 <1512000000>;
 		opp-microvolt = <1075000>;
 	};
 };
 
 &cluster1_opp {
-	opp00 {
+	opp@408000000 {
 		opp-hz = /bits/ 64 <408000000>;
 		opp-microvolt = <800000>;
 		clock-latency-ns = <40000>;
 	};
-	opp01 {
+	opp@600000000 {
 		opp-hz = /bits/ 64 <600000000>;
 		opp-microvolt = <800000>;
 	};
-	opp02 {
+	opp@816000000 {
 		opp-hz = /bits/ 64 <816000000>;
 		opp-microvolt = <800000>;
 	};
-	opp03 {
+	opp@1008000000 {
 		opp-hz = /bits/ 64 <1008000000>;
 		opp-microvolt = <850000>;
 	};
-	opp04 {
+	opp@1200000000 {
 		opp-hz = /bits/ 64 <1200000000>;
 		opp-microvolt = <925000>;
 	};
-	opp05 {
+	opp@1416000000 {
 		opp-hz = /bits/ 64 <1416000000>;
 		opp-microvolt = <1025000>;
 	};
-	opp06 {
+	opp@1608000000 {
 		opp-hz = /bits/ 64 <1608000000>;
 		opp-microvolt = <1125000>;
 	};
-//	opp07 {
-//		opp-hz = /bits/ 64 <1800000000>;
-//		opp-microvolt = <1200000>;
-//	};
+	opp@1800000000 {
+		opp-hz = /bits/ 64 <1800000000>;
+		opp-microvolt = <1200000>;
+		status = "disabeld";
+	};
 };
 
 &CPU_COST_A72 {
diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index ffa7197611b1..019e3c5b8417 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -198,28 +198,28 @@
 		compatible = "operating-points-v2";
 		opp-shared;
 
-		opp00 {
+		opp@408000000 {
 			opp-hz = /bits/ 64 <408000000>;
 			opp-microvolt = <800000>;
 			clock-latency-ns = <40000>;
 		};
-		opp01 {
+		opp@600000000 {
 			opp-hz = /bits/ 64 <600000000>;
 			opp-microvolt = <800000>;
 		};
-		opp02 {
+		opp@816000000 {
 			opp-hz = /bits/ 64 <816000000>;
 			opp-microvolt = <800000>;
 		};
-		opp03 {
+		opp@1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
 			opp-microvolt = <875000>;
 		};
-		opp04 {
+		opp@1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
 			opp-microvolt = <925000>;
 		};
-		opp05 {
+		opp@1416000000 {
 			opp-hz = /bits/ 64 <1416000000>;
 			opp-microvolt = <1025000>;
 		};
@@ -229,24 +229,24 @@
 		compatible = "operating-points-v2";
 		opp-shared;
 
-		opp00 {
+		opp@408000000 {
 			opp-hz = /bits/ 64 <408000000>;
 			opp-microvolt = <800000>;
 			clock-latency-ns = <40000>;
 		};
-		opp01 {
+		opp@600000000 {
 			opp-hz = /bits/ 64 <600000000>;
 			opp-microvolt = <800000>;
 		};
-		opp02 {
+		opp@816000000 {
 			opp-hz = /bits/ 64 <816000000>;
 			opp-microvolt = <800000>;
 		};
-		opp03 {
+		opp@1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
 			opp-microvolt = <850000>;
 		};
-		opp04 {
+		opp@1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
 			opp-microvolt = <925000>;
 		};
@@ -1254,15 +1254,15 @@
 		compatible = "operating-points-v2";
 		opp-shared;
 
-		opp00 {
+		opp@200000000 {
 			opp-hz = /bits/ 64 <200000000>;
 			opp-microvolt = <900000>;
 		};
-		opp01 {
+		opp@300000000 {
 			opp-hz = /bits/ 64 <300000000>;
 			opp-microvolt = <900000>;
 		};
-		opp02 {
+		opp@400000000 {
 			opp-hz = /bits/ 64 <400000000>;
 			opp-microvolt = <900000>;
 		};
-- 
2.35.3

